script/after_failure.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Maybe we can put those scripts in `dev/ci` instead of `script` as we already have a lot of things at the root of the repository